processor/src/stack/mod.rs

```diff
@@ -140,22 +127,23 @@ impl Stack {
         result
     }
 
-    /// Returns [StackOutputs] consisting of all values on the stack and all addresses in the
-    /// overflow table that are required to rebuild the rows in the overflow table.
-    pub fn build_stack_outputs(&self) -> StackOutputs {
+    /// Returns [StackOutputs] consisting of all values on the stack.
+    pub fn build_stack_outputs(&self) -> Result<StackOutputs, ExecutionError> {
```

Comment:
Let's add `Errors` section to the doc comments to explain under which condition this can fail.